### PR TITLE
Update/add images for brats_mri_segmentation

### PR DESCRIPTION
**Pull request by myron**
Update/add images for brats_mri_segmentation

### CODE REVIEW

**Comment by yiheng-wang-nv** [comment]
This PR is to fix #222

**Comment by Nic-Ma** [comment]
Hi @myron ,

Do you also want to update the `large_files` in this PR?
When you update any thing in a bundle, please ensure to update the `changelog` and `version` of `metadata.json`.

Thanks.

File: models/brats_mri_segmentation/docs/README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ The model is trained to segment 3 nested subregions of primary brain tumors (gli
 - The TC describes the bulk of the tumor, which is what is typically resected. The TC entails the ET, as well as the necrotic (fluid-filled) and the non-enhancing (solid) parts of the tumor.
 -  The WT describes the complete extent of the disease, as it entails the TC and the peritumoral edema (ED), which is typically depicted by hyper-intense signal in FLAIR.
 
+![](https://developer.download.nvidia.com/assets/Clara/Images/clara_pt_brain_mri_segmentation_workflow.png)
+
```
Comment (yiheng-wang-nv) [inline models/brats_mri_segmentation/docs/README.md:12]: Thanks @myron . Please also update the changelog and version number in metadata.json
and put new weights download link into large_files.yml

Comment (myron) [reply models/brats_mri_segmentation/docs/README.md:12]: @yiheng-wang-nv  I've updated the changelog and version number.  based on the conversation now, we decided to keep the existing weights as is. please review and merege

**Comment by myron** [comment]
> Hi @myron ,
> 
> Do you also want to update the `large_files` in this PR? When you update any thing in a bundle, please ensure to update the `changelog` and `version` of `metadata.json`.
> 
> Thanks.

I've updated the changelog and version. and we're keeping existing weights.

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**